graftm/run.py

```diff
@@ -367,26 +363,24 @@ def graft(self):
                                self.args.output_directory,
                                False)
 
+
+        # Classification steps        
```

Comment:
shouldn't this be inside the next `if` statement? Or so you want to cluster before diamond as well?